### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by cjyabraham** [LF-Engineering/lfevents#261]
from @thetwopct :

Alt Attributes
Pull in alt attributes for images, or write in for images in theme files. A few missing on pages here and there (Linux Foundation logo)

Name buttons and links
Using conventions like aria-label="Name" values on buttons (i.e. mobile Menu), or aria-label="Read more about X" on links without discernible name (i.e. Read more)
Example on front page - “Register” link should have an aria-label=”Register for X event”

Add description to Video Block
Add a small description to video embeds iFrame

**Comment by thetwopct** [comment]
Just had a quick look at this

- Some contrast issues with color choices - we'll need to see if new brand guidelines addresses these
- The footer had some issues with links not being obvious (even though they were bolded), I added some underlines, and also updated the grey a touch to be inline with the style guide colors.
- Some sections needed a "role" added to them to better identify them to screen readers.
- Search Filter needed screen reader text added (keep in my mind for other LF sites that use this plugin)
![2021-01-16-145932@2x](https://user-images.githubusercontent.com/10615884/104815285-825dc500-580b-11eb-8cda-da1c3c5da9bd.jpg)

**Comment by cjyabraham** [comment]
I've split out another ticket to remind us to [set screen reader text for cncf and lfph search/filter displays](https://github.com/cncf/cncf.io/issues/375).
